subsys/bluetooth/mesh/scheduler_srv.c

```diff
@@ -554,6 +568,8 @@ static int send_scheduler_action_status(struct bt_mesh_model *model,
 	struct bt_mesh_scheduler_srv *srv = model->user_data;
 
 	BT_MESH_MODEL_BUF_DEFINE(buf, BT_MESH_SCHEDULER_OP_ACTION_STATUS,
+		srv->sch_reg[idx].action == BT_MESH_SCHEDULER_NO_ACTIONS ?
```

Comment:
This is not fully correct. When Scheduler Action Status is sent in response to Scheduler Action Set, the Scheduler Register field should be the same as a request message.